probe/endpoint/conntrack_test.go

```diff
@@ -0,0 +1,138 @@
+package endpoint_test
+
+import (
+	"bufio"
+	"encoding/xml"
+	"io"
+	"testing"
+	"time"
+
+	. "github.com/weaveworks/scope/probe/endpoint"
```

Comment:
Yeah I think this is the one acceptable place for a dot import. In a separate PR I'll update the other tests to use this style. 

https://github.com/golang/go/wiki/CodeReviewComments#import-dot